src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -158,6 +158,21 @@ function updateStatsCheckBox() {
     $('input[id=statsShownCheckBox]').val($('#show-stats-checkbox').is(':checked'));
 }
 
+function printView() {
+    // Fix to hide the filter placeholder when it is empty.
+    if ($('#results-search-box').val()) {
+        $('#results-search-box').parent().parent().removeClass('hide-for-print');
+    } else {
+        $('#results-search-box').parent().parent().addClass('hide-for-print');
+    }
+    
+    $('.container').printThis({
```

Comment:
Not so fond of using the class as the selector; it might be better to use the id `mainContent` instead, unless there are reasons to use `.container` specifically, as I noticed that the navbar and footer have class `container` as well.